src/encord_active/app/common/page.py

```diff
@@ -29,38 +31,75 @@ def __repr__(self):
         return f"{type(self).__name__}()"
 
     @staticmethod
-    def row_col_settings_in_sidebar():
+    def display_settings(metric_scope: MetricScope):
         col_default_max, row_default_max = 10, 5
 
+        global_state = get_state()
         with st.expander("Settings"):
             setting_columns = st.columns(3)
 
-            get_state().page_grid_settings.columns = int(
-                setting_columns[0].number_input(
-                    "Columns",
-                    min_value=2,
-                    max_value=col_default_max,
-                    value=State.page_grid_settings.columns,
-                    help="Number of columns to show images in the main view",
+            with setting_columns[0]:
+                get_state().page_grid_settings.columns = int(
+                    st.number_input(
+                        "Columns",
+                        min_value=2,
+                        max_value=col_default_max,
+                        value=State.page_grid_settings.columns,
+                        help="Number of columns to show images in the main view",
+                    )
                 )
-            )
+                if metric_scope == MetricScope.DATA_QUALITY:
+                    st.markdown("")
+                    st.markdown("")
+                    global_state.object_drawing_configurations.draw_objects = st.checkbox(
+                        "Show object labels",
+                        value=ObjectDrawingConfigurations.draw_objects,
+                        help="If checked, bounding boxes and polygons will be shown on the images.",
+                    )
+                else:
+                    global_state.object_drawing_configurations.draw_objects = True
```

Comment:
Why 'show object labels' checkbox is not optional in label quality pages?

Reply:
Because it makes no sense to not see the labels when you're in the label page? .. At least, that was my thinking